poc/test_opaque_ake.sage

```diff
@@ -75,6 +76,7 @@ def test_core_protocol_serialization():
     skS, pkS = group.key_gen()
     skS_enc = group.serialize_scalar(skS)
     pkS_enc = group.serialize(pkS)
+    oprf_seed = random_bytes(config.oprf_suite.group.scalar_byte_length())
```

Comment:
I believe this should be `Nh` (length of the hash output) bytes long, because `oprf_seed` is fed into the KDF's expand function.